src/OutSystems/Maps/MapAPI/MarkerManager.ts

```diff
@@ -47,6 +47,67 @@ namespace OutSystems.Maps.MapAPI.MarkerManager {
         if (marker.hasPopup) marker.refreshPopupContent();
     }
 
+    /**
+     * Function that will remove a map marker with a given position from a cluster.
+     *
+     * @param mapId Id of the Map
+     * @param markerPosition Defines the location of the marker. Works with addresses and coordinates (latitude and longitude).
+     */
+    export function RemoveMarkerFromCluster(
+        mapId: string,
+        markerPosition: string
+    ): string {
+        const responseObj = {
+            isSuccess: true,
+            message: 'Success',
```

Comment:
Should use ErrorCodes.SuccessMessage and ErrorCodes.SuccessCode

Reply:
@BenOsodrac yap, but on a different interaction since this is the only one API method that contains this approach yet, and on top of that we must also update the ErrorCodes into constants. That's why I didn't apply it on this scope.